code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -226,9 +226,14 @@
 	return can_unwrench
 
 // Throws the user when they unwrench a pipe with a major difference between the internal and environmental pressure.
-/obj/machinery/atmospherics/proc/unsafe_pressure_release(mob/user, pressures = null)
+/obj/machinery/atmospherics/proc/unsafe_pressure_release(mob/living/carbon/user, pressures = null)
 	if(!user)
 		return
+	if(ishuman(user)) //other carbons like monkeys can unwrench but cant wear magboots
+		if(istype(user.shoes, /obj/item/clothing/shoes/magboots))
```

Comment:
check if their shows exist?

Reply:
damn

Reply:
```if( user.shoes &&  istype(user.shoes, /obj/item/clothing/shoes/magboots))```

Reply:
actually, i dont get a runtime, its not necessary